### PR TITLE
Remove Ruff

### PR DESCRIPTION
**Pull request by Totto16**
Remove ruff, since it's not needed

### CODE REVIEW

File: lobby/main.py [hunk 1]
```diff
@@ -1,24 +1,19 @@
 import os.path
 import time
 import typing
-from dataclasses import dataclass
-from dataclasses import field
+from dataclasses import dataclass, field
```
Comment (mgerhold) [inline lobby/main.py]: I don't want collapsed imports.

File: lobby/main.py [hunk 2]
```diff
@@ -48,41 +43,41 @@ class Lobby:
...
-    def from_id(cls, id_: str) -> Self:
+    @staticmethod
+    def from_id(id_: str) -> "PlayerInfo":
         user = typing.cast(Optional[User], User.query.filter_by(id=id_).first())
         if user is None:
-            raise KeyError(f"User with id {id_} not found")
+            raise KeyError(f"User with id {id_} not found")  # noqa: TRY003, EM102
         assert isinstance(user, User)
-        return cls(id_, user.username)
+        return PlayerInfo(id_, user.username)
```
Comment (mgerhold) [inline lobby/main.py]: What's the purpose of this change? I'm confused!

**Review by mgerhold — CHANGES_REQUESTED** [review]
All in all I don't like these changes. Either I re-add ruff myself, or you rework this somehow (which could be difficult if you don't know how I want it to be). Oooor we remove ruff all together ¯\\\_(ツ)_/¯

**Comment by Totto16** [comment]
The best solution is to remove it 👍🏼

**Review by mgerhold — APPROVED** [review]
Thanks! Gonna merch this even though the CI fails. I have to rework the stuff to make pyright work again, but that's not on your side :)
